dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeAPIImpl.java

```diff
@@ -39,24 +40,24 @@
 
 public class ContentTypeAPIImpl implements ContentTypeAPI {
 
-  final ContentTypeFactory fac;
-  final FieldFactory ffac;
-  final PermissionAPI perms;
-  final User user;
-  final Boolean respectFrontendRoles;
-  final FieldAPI fAPI;
+  private final ContentTypeFactory contentTypeFactory;
+  private final FieldFactory fieldFactory;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "fieldFactory" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)